Homework3/Homework3/Homework3/Program.cs

```diff
@@ -1,4 +1,6 @@
-﻿//Console.WriteLine("input number");
+﻿//Problem #1----
+
+//Console.WriteLine("input number");
 //int number;
 //number = Convert.ToInt32(Console.ReadLine());
 
```

Comment:
ეს ვერ დაწერე თუ გამოგრჩა ?

Reply:
აუ, კომენტარების მოხსნის დროს წამეშალა, მივაწერ თავიდან

Reply:
მოიცა, პირველ პრობლემაზე წერ? ეგ მიწერია

Reply:
მეორეზე გამრავლება და გაყოფა არაა არ მივაქციე ყურადღება რომელ ლაინზე დავსვი

Reply:
ვნახე გამრავლებაც 
if(num1==0 || num2 ==0){
Console.writeline("you can't divide on 0");
}
და  მერე იგივე შემოწმება როგორც გამოკლებაზე დიდი რიცხვი რომ იპოვო და  გამოთვლიდი